### PR TITLE
Merge release 15.9 into trunk

### PR DESCRIPTION
**Pull request by loremattei**
Merge release 15.9 into trunk

PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/metadata/PlayStoreStrings.po [hunk 1]
```diff
@@ -11,21 +11,20 @@ msgstr ""
...
-msgctxt "release_note_157"
+msgctxt "release_note_158"
 msgid ""
-"15.7:\n"
-"Improvements in the Reader: A new Discover tab you can tailor to your interests and an “(un)follow” button to the site and topic detail screens.\n"
+"15.8:\n"
+"- Block editor enhancements: Added full-width and wide alignment support for Group, Cover, and Image blocks; added support for the Rounded style in the Image block; and resolved an issue where the toolbar behaved unexpectedly when adding a block.\n"
+"- Reader update: Preview images are now displayed on posts when no featured image is set.\n"
 "\n"
-"Block Editor enhancements: A new Pullquote block that lets you add emphasis to a chosen piece of text. Any changes made in menu sliders are now also immediately reflected in block settings.\n"
```
Comment (AliSoftware) [inline WordPress/metadata/PlayStoreStrings.po:28]: Is it expected that `release_note_157` was removed? Like, do we only keep the translations for only the last 2 release versions in GlotPress?

File: gradle.properties-example [hunk 2]
```diff
@@ -41,6 +41,7 @@ wp.e2e.wp_com_user.email=e2eflowtestingmobile@example.com
 wp.e2e.wp_com_user.password=mocked_password
 wp.e2e.wp_com_user.site_address=e2eflowtestingmobile.wordpress.com
 wp.e2e.wp_com_user.username=e2eflowtestingmobile
+wp.e2e.wp_com_passwordless_user.email=e2eflowtestingmobile+passwordless@example.com
```
Comment (AliSoftware) [inline gradle.properties-example:44]: Unrelated question: How do we inform devs when changes are made in the properties example files like here – as I'm guessing they are expected to transfer those additions/changes to their local `.properties` files on their machine?

**Review by AliSoftware — APPROVED** [review]
Couldn't smoke-test it on device, but diff lgtm

(left some questions for my own culture)

File: WordPress/metadata/PlayStoreStrings.po (hunk 1 above)
Comment (loremattei) [reply WordPress/metadata/PlayStoreStrings.po:28]: Yep, right!

**Pull request merged**